analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/RuleTypeMappingCS.cs

```diff
@@ -0,0 +1,6928 @@
+﻿/*
```

Comment:
I tried to do it as a rename in a separate commit, but git didn't persisted that here